### DISCUSSION TITLE
How to enable single Click on Thumbnail to load images into main canvas

### DISCUSSION THREAD

**Discussion by arunjoshtech** [nroduit/Weasis#529] [General]
Dear folks,
 I want to enable single click on thumbnail to load images in to main window instead of double click on it. Can anyone guide me how to enable it please

**Comment by nroduit** [comment]
A single click is already used for selection to open/add/delete/merge several series.

Otherwise, to change series more quickly, there have been recent improvements in patient/exam/series navigation, see in [Tip window](https://weasis.org/en/tutorials/dicom-explorer/).
